src/platform/plugins/shared/discover/public/context_awareness/EXTENSION_POINTS_INVENTORY.md

```diff
@@ -0,0 +1,270 @@
+
+**Changes:** Flyout title and tabs (add/modify).
+
+**Use when:** Add "Related events", "Stack trace", "Analysis".
+
+**Returns:** `{ title?, docViewsRegistry(prev) => DocViewsRegistry }`
+
+**Screenshot:**
+![Doc viewer example](./docs/extension_points/get_doc_viewer.png)
+
+**Implementation examples:**
+- [Log document profile](profile_providers/observability/log_document_profile/accessors/get_doc_viewer.tsx)
+- [Traces document profile](profile_providers/observability/traces_document_profile/document_profile/accessors/doc_viewer.tsx)
+- [Observability document profile](profile_providers/observability/observability_document_profile/document_profile/accessors/doc_viewer.tsx)
+- [Observability root profile](profile_providers/observability/observability_root_profile/accessors/get_doc_viewer.tsx)
+
+---
+
+## Top nav
+
+### App menu — `getAppMenu`
+
+**Changes:** Up to two custom top‑nav actions.
+
+**Use when:** Export, open custom app, open new tab, alerting shortcuts.
+
+**Returns:** `{ appMenuRegistry(prev) => AppMenuRegistry }` (params include `dataView`, `adHocDataViews`, `isEsqlMode`, `authorizedRuleTypeIds`)
+
+**Screenshot:**
+![App menu example](./docs/extension_points/get_app_menu.png)
+
+**Implementation examples:**
+- [Observability root profile](profile_providers/observability/observability_root_profile/accessors/get_app_menu.tsx)
+
+---
+
+## Columns
+
+### Columns configuration — `getColumnsConfiguration`
+
+**Changes:** Header labels, icons, and metadata for columns.
+
+**Use when:** Rename headers, add icons, improve tooltips.
+
+**Returns:** `CustomGridColumnsConfiguration`
+
+**Screenshot:**
+![Columns configuration example](./docs/extension_points/get_columns_configuration.png)
+
+**Implementation examples:**
+- [Logs data source profile](profile_providers/observability/logs_data_source_profile/accessors/get_columns_configuration.tsx)
+- [Traces data source profile](profile_providers/observability/traces_data_source_profile/accessors/get_columns_configuration.tsx)
+
+---
+
+## Field list
+
+### Recommended fields — `getRecommendedFields`
+
+**Changes:** "Recommended" section in the Field List.
+
+**Use when:** Surface domain‑critical fields.
+
+**Returns:** `{ recommendedFields: string[] }`
+
+**Screenshot:**
+![Recommended fields example](./docs/extension_points/get_recommended_fields.png)
+
+**Implementation examples:**
+- [Logs data source profile](profile_providers/observability/logs_data_source_profile/accessors/get_recommended_fields.ts)
+
+---
+
+## Cell actions integration
+
+### Trigger & metadata: `DISCOVER_CELL_ACTIONS_TRIGGER`, `DiscoverCellAction`
+
+**Changes:** Standardizes how cell actions receive context (`instanceId`, `dataSource`, `dataView`, `query`, `filters`, `timeRange`).
+
+**Use when:** Build cell‑action plugins or cross‑app actions.
+
+**Returns:** `CellAction`
+
+**Implementation examples:**
+- See [types.ts](./types.ts) for `DISCOVER_CELL_ACTIONS_TRIGGER` and `DiscoverCellAction` definitions
```

Comment:
generally looking pretty good and is definitely useful 

one thing I wonder, we've got the getAdditionalCellActions described before, so is this `## Cell actions integration` necessary, since as implementation examples is just referring `types.ts`?

Reply:
Yeah I can see how it is a duplicate. Removed in https://github.com/elastic/kibana/pull/246369/commits/e1039fa0fbad86ab6b23e5f78cfcb59153d1cc9c